wsdl/ver10/schema/common.xsd

```diff
@@ -83,6 +83,23 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 			</xs:element>
 		</xs:sequence>
 	</xs:complexType>
+	<xs:complexType name="FieldOfView">
+		<xs:attribute name="hfov" type="xs:float" use="required">
+			<xs:annotation>
+				<xs:documentation>Horizontal field-of-view in degrees.</xs:documentation>
+			</xs:annotation>
+		</xs:attribute>
+		<xs:attribute name="vfov" type="xs:float" use="required">
+			<xs:annotation>
+				<xs:documentation>Vertical field-of-view in degrees.</xs:documentation>
+			</xs:annotation>
+		</xs:attribute>
+		<xs:attribute name="mag" type="xs:float">
+			<xs:annotation>
+				<xs:documentation>Magnification compared to zoomed-out state. When fully zoomed out equals 1.0.</xs:documentation>
+			</xs:annotation>
+		</xs:attribute>
```

Comment:
As noted below, this should just change into a FOV space instead of including mag.

Reply:
ack.